src/tests/electron/main.test.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Electron from 'electron';
+import { Application } from 'spectron';
+
+describe('Electron E2E', () => {
+    let app: Application;
+
+    beforeAll(() => {
+        const electronPath = `${(global as any).rootDir}/drop/electron/extension/bundle/main.bundle.js`;
+        app = new Application({
+            path: Electron as any,
```

Comment:
Why the "as any" here?

Reply:
From Spectron's [docs](https://github.com/electron-userland/spectron#application-api):

> If you want to invoke electron directly with your app's main script then you should specify path as electron via electron-prebuilt and specify your app's main script path as the first argument in the args array.

electron-prebuilt is now just electron. The typings don't line up because in all other use cases, you would actually specify a string path. I felt like following the official guidance was worth the `as any` in this case. It's possible our future packaging work will change this if we use a different binary.